composer.json

```diff
@@ -43,9 +43,9 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.5-dev"
+            "dev-master": "1.1-dev"
```

Comment:
Well, @dbu came with the same reason. I thought otherwise so far, but this is not something which I cared about too much, so probably you are right.